<launch>

  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen">
    <param name="num_worker_threads" value="10" />
  </node>

  <node pkg="nodelet" type="nodelet" name="image_synchroniser" args="load sl_sensor_image_acquisition/ImageSynchroniserNodelet standalone_nodelet" output="screen">
    <param name="projector_topic" type="string" value="/versavis/projector/image_time" />
    <param name="output_topic" type="string" value="/image_synchroniser/grouped_images" />
    <param name="frame_id" type="string" value="sl_sensor" />
    <param name="lower_bound_tol" type="double" value="0.0166667" />
    <param name="upper_bound_tol" type="double" value="0.0166667" />
    <param name="image_trigger_period" type="double" value="0.03333333" />
    <param name="projector_yaml_directory" type="string" value="$(find sl_sensor_projector)/projector_yaml/lightcrafter4500.yaml"/>
    <param name="projector_service_name" type="string" value="/command_projector"/>    
    <param name="image_topics" type="string" value="/versavis/cam1/image_raw" />
  </node>		  

  <node pkg="nodelet" type="nodelet" name="image_array_logger" args="load sl_sensor_image_acquisition/ImageArrayLoggerNodelet standalone_nodelet" output="screen">
    <param name="log_directory" type="string" value="/home/ltf/Desktop/logged_data/" />
    <param name="file_header_" type="string" value="test" />
    <param name="log_topic" type="string" value="/image_synchroniser/grouped_images" />
  </node>

</launch>
